backend/src/main/java/com/example/backend/pathfinding/Graph.java

```diff
@@ -0,0 +1,28 @@
+package com.example.backend.pathfinding;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.Map;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Graph {
+    private Map<String,Vertex> vertices;
+    private List<Edge> edges;
+    public void addVertex(String vertexName){
+        vertices.put(vertexName,new Vertex(vertexName,0,""));
+    }
+
+    public void addNonDirectedEdge(Edge edge){
```

Comment:
Das brauchen wir für unseren Algorithmus eigentlich auch nicht, oder? 
In `DistanceMatrix.toGraph` wird ja `addDirectedEdge` benutzt.

Reply:
Agreed*